Algebras/DotBracket/Parts/algpart_dotBracket_basic.gap

```diff
@@ -149,5 +148,5 @@
 
   choice [string] h([string] i) {
     //~ return list(minimum(i));
-    return i;
```

Comment:
changing from ID, i.e. just return `i` might brake things at other places, thus I think it would be better to create another dotBracket algebra, which inherits all but the choice function. Add the following lines to `Algebras/DotBracket/alg_dotBracket.gap` and make sure to use the new algebra for gapc compilation.
```
algebra alg_dotBracket_unique extends alg_dotBracket {
	choice [string] h([string] i) {
	return unique(i);
}

Reply:
I think this is still current. Can you please revert this file, the new algebra with `unique` looks good in the main alg_dotBrackt file - but remember to change in your test file